spec.emu

```diff
@@ -1262,6 +1262,23 @@ location: https://github.com/tc39/proposal-decimal/
       1. Throw a *TypeError* exception.
     </emu-alg>
   </emu-clause>
+
+  <emu-clause id="sec-decimal128.prototype.withfractionaldigits">
+    <h1>Decimal128.prototype.withFractionalDigits ( _digits_ )</h1>
+    <p>This method performs the following steps when called:</p>
+    <emu-alg>
+      1. Let _O_ be the *this* value.
+      1. Perform ? RequireInternalSlot(_O_, [[Decimal128Data]]).
+      1. If _digits_ is not a Number, throw a *TypeError* exception.
+      1. If _digits_ is *NaN*<sub>𝔽</sub>, *+∞<sub>𝔽</sub>*, or *+∞<sub>𝔽</sub>*, throw a *RangeError* exception.
+      1. If ℝ(_digits_) is not an integer, throw a *TypeError* exception.
+      1. IF ℝ(_digits_) < 0, throw a *RangeError* exception.
```

Comment:
The name `.withFractionalDigits` might not be ideal, but it'd be great to allow negative numbers here.

`1234..withFractionalDigits(1)` is `1234.0`, and `1234..withFractionalDigits` is `123e1`.

Let's see if this is desired though. If nobody cares it's fine to throw for simmetry with `.toFixed()`.